resources/public/api/conf/2.0/testdata/fund-release-extension.md

```diff
@@ -533,6 +534,7 @@
 {
   "code": "FUND_RELEASE_SUPERSEDED",
   "message": "This fund release has already been superseded"
+  "lifeEventId": "3456789001"
```

Comment:
This is perfect, but in reviewing this I now release the fund release id in the request body needs to be 3456789000. Sorry!